lego/apps/events/serializers/events.py

```diff
@@ -74,7 +74,7 @@ class Meta:
             'comment_target', 'start_time', 'end_time', 'merge_time', 'pools',
             'unregistration_deadline', 'company', 'active_capacity', 'feedback_description',
             'feedback_required', 'is_priced', 'price_member', 'price_guest', 'use_stripe',
-            'payment_due_date', 'use_captcha', 'waiting_registrations', 'tags', 'is_merged',
+            'payment_due_date', 'use_captcha', 'waiting_registration_count', 'tags', 'is_merged',
```

Comment:
This serializer also needs `is_abakom_only`, in order to retain the value when opening the eventEditor :smile: